submitted_models/marble_husky_sensor_config_3/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>marble_husky_sensor_config_3</name>
+  <version>0.1.0</version>
+  <description>The marble_husky_sensor_config_3 package
+    - Has two rgbd camera Realsense D435
```

Comment:
There is only one RGBD camera in the SDF file

Reply:
You are correct. The package.xml has been updated to more accurately reflect the vehicle configuration.